packages/react/src/PageHeader/PageHeader.dev.stories.tsx

```diff
@@ -18,11 +18,11 @@ export default meta
 
 export const LargeVariantWithMultilineTitle = () => (
   <Box sx={{padding: 3}}>
-    <PageHeader>
+    <PageHeader titleVariant="large">
```

Comment:
I feel good about this API change. Even though, it seems more title related, we also manage the `height` of the all these leading and trailing visuals and actions by this prop. Because when title font size, line height etc is changing depending on the size, the height of the other elements change too. So I think it makes better sense to me to have this prop higher in the hierarchy. 

Also, open to naming suggestions 🙌🏻